klaus/contrib/auto_klaus.py

```diff
@@ -0,0 +1,103 @@
+"""
+Alternative take on the "automatically discovered repositories" concept
+that requires no threads, polling or inotify. Instead the filesystem is
+consulted whenever a repository name is looked up.
```

Comment:
I feel like it's important to mention here that each time the repo list is viewed by someone, the following code is run:

- 1x `listdir()` in the root
- Nx `os.path.exists` in each of the folders

Reply:
Added a blurb to that effect to the `wsgi_autodetecting.py` docstring.